src/styles/styles.css

```diff
@@ -1,3 +1,7 @@
-.MuiBox-root input[type="search"] {
-  appearance: textField !important;
+html {
+  overflow: auto;
+}
+
+body {
+  font-family: "Open Sans";
```

Comment:
We should be using Poppins everywhere. If the design is using Open Sans, we must correct the design

Reply:
ok